src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine[] machines = new Machine[3];
+        machines[0] = new Truck();
+        machines[1] = new Bulldozer();
+        machines[2] = new Excavator();
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+
```

Comment:
a bit of lines formatting and you are good to go :)
```suggestion
        Machine[] machines = new Machine[3];
        machines[0] = new Truck();
        machines[1] = new Bulldozer();
        machines[2] = new Excavator();
        for (Machine machine : machines) {
            machine.doWork();
            machine.stopWork();
        }
```